File: backend/test-login.ts
import "dotenv/config";
import fetch from "node-fetch"; // or just use native fetch if Node >= 18

async function main() {
  const res = await fetch('http://localhost:3000/api/auth/login', {
    method: 'POST',
    headers: { 'Content-Type': 'application/json' },
    body: JSON.stringify({ email: 'admin@nala.com', password: 'password123' })
  });
  const text = await res.text();
  console.log('Status:', res.status);
  console.log('Response:', text);
}
main();

File: mobile/test_login.dart
import 'dart:convert';
import 'package:http/http.dart' as http;

void main() async {
  try {
    final response = await http.post(
      Uri.parse('http://127.0.0.1:3001/api/auth/login'),
      headers: {'Content-Type': 'application/json'},
      body: jsonEncode({'email': 'admin@nala.com', 'password': 'password123'}),
    );
    print('Status: ${response.statusCode}');
    print('Body: ${response.body}');
  } catch (e) {
    print('Error: $e');
  }
}
